src/tests/test_tls_signature_scheme.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+* (C) 2022 Jack Lloyd
+* (C) 2022 Hannes Rantzsch, René Meusel - neXenio
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "tests.h"
+
+#if defined(BOTAN_HAS_TLS)
+
+#include <botan/tls_signature_scheme.h>
+
+#include <botan/ec_group.h>
+
+namespace {
+
+using namespace Botan::TLS;
```

Comment:
Remove this and use `Botan::TLS` prefix explicitly. Or if worst case it is truly too verbose for some reason, contain the using namespace to within a function.